ludwig/datasets/configs/naval.yaml

```diff
@@ -28,9 +28,5 @@ description: |
   Condition Based Maintenance of Naval Propulsion Plants Data Set
   http://archive.ics.uci.edu/ml/datasets/condition+based+maintenance+of+naval+propulsion+plants
 output_features:
-  - name: lp
-    type: number
-  - name: gtcdsc
-    type: number
-  - name: gttdsc
-    type: number
+- name: gtcdsc
```

Comment:
Interesting. I specified it that way to be consistent with @anneholler's work here: https://github.com/ludwig-ai/experiments/blob/main/automl/validation/naval/auto_config.json.automl
I suggest we keep the features consistent since they're tied to blog posts.